Makefile

```diff
@@ -208,6 +208,10 @@ cleanup-test-cockroach:
 test-e2e:
 	test/docker_e2e.sh
 
+.PHONY: hygiene
+hygiene:
+	test/repo_hygiene/repo_hygiene.sh
```

Comment:
Yes, currently I think that makes sense.  But, I was thinking this might go beyond just lint checks at some point which was the motivation for making it separate.  I'm not sure that assumption was justified, but I agree that this would be included in the things we should resolve in an improved wholistic approach to CI -- I've opened #894 to track this.